### ISSUE TITLE
Tried to attach picture to unapproved chat, image was sent, but recipient reported can't see it after approve.

### ISSUE THREAD

**Issue by SoNickRND** [oxen-io/session-desktop#2389]
Tried to attach picture to unapproved chat, image was sent, but recipient reported can't see it after approve.
I think, if it's the same policy "no binary exchange without approve", then:
1) I shouldn't be able to send image with same error message "you can't send files untill you get approve"
or
2) Recipient should be able to see my images send prior to approve.

**Comment by SoNickRND** [comment]
Mine was win7 64bit desktop, dunno what was on other side.
(Winver says Version 6.1 build 7601: SP1)

**Comment by Freeplayg** [comment]
I had sort of the same thing by dragging an image in, but it wouldn't let me send the image and wouldn't give a reason why. So I sort of just thought that it broke.

On Fedora 36, using the Flatpak from Gnome Software

**Comment by KeeJef** [comment]
Yeah this should always fail and should tell the user why its failing (the recipient has not yet accepted the message request)

**Comment by Bilb** [comment]
the root issue is that the Alice was able to send an image to Bob without being approved by Bob. I believe we lock the UI for that, but we could also lock the send logic.

in `ts/models/conversation.ts` 
`  public async sendMessage(msg: SendMessageType) {`
before calling `addSingleOutgoingMessage` we should make sure that the conversation is approved (if this is a private conversation) and if not, show a toast and return early.
